js/main.js

```diff
@@ -164,14 +164,16 @@ function viewSwapDataViews(dataView) {
     document.forms[0].location.value = data.profile.location;
     document.forms[0].bio.value = data.profile.bio;
 
-    imagePlaceholder.src = data.profile.avatarUrl;
+    if (data.profile.username !== '') {
+      imagePlaceholder.src = data.profile.avatarUrl;
```

Comment:
Don't you want this to be setting the url for the non-placeholder image? If I'm understanding this right, if the user does have a profile image, you aren't setting the `src` for the profile image, you are overwriting the placeholder img `src`.